packages/components/src/core/styles/common/types.ts

```diff
@@ -97,6 +97,20 @@ declare module "@mui/material/styles" {
   export interface PaletteOptions {
     sds?: SDSPalette;
   }
+
+  /**
+   * @see https://mui.com/material-ui/customization/breakpoints/#custom-breakpoints
+   * (masoudmanson): ModuleAugmentation is required for TypeScript to recognize the
+   * custom breakpoints added to the theme.
+   */
+  export interface BreakpointOverrides {
+    // remove the xs and xl breakpoints since SDS does not support them
```

Comment:
awesome comment! Thank you 👏

Reply:
Thanks so much for taking the time to go through the PRs! Really grateful for your time and feedback! You're amazing 😍🥳🎉✨

Reply:
Always a pleasure, Masoud!! Thank you again for your continued outstanding contribution to SDS 🤩 🔥 ✨🙏 👏  !!